.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: node
+node_js:
+- 14.15.1
+addons:
+  ssh_known_hosts:
+  - 134.122.39.75
+branches:
+  only:
+  - main
+  - ci_cd_liwg
+script:
+- openssl aes-256-cbc -K $encrypted_b0d51b9ff91b_key -iv $encrypted_b0d51b9ff91b_iv -in deploy.enc -out deploy -d
+- chmod 600 deploy && mv deploy ~/.ssh/id_rsa
+deploy:
+- provider: script
+  skip_cleanup: true
```

Comment:
`skip_cleanup` is now deprecated, and `cleanup` is `false` by default. The default used to be true, so you had to opt out using `skip_cleanup` … and has been used a lot. Cleaning up the working directory from any left over build artifacts only made sense for few providers, so we have changed this default.

**Reference:** https://blog.travis-ci.com/2019-08-27-deployment-tooling-dpl-v2-preview-release